bvm/ballerina-core/src/main/java/org/ballerinalang/connector/api/ConnectorUtils.java

```diff
@@ -98,17 +95,4 @@ public static BallerinaServerConnector getBallerinaServerConnector(Context conte
         ProgramFile programFile = context.getProgramFile();
         return getBallerinaServerConnector(programFile, protocolPkgPath);
     }
-
-    /**
-     * This method is used to convert a JSON to struct given the resource, json and required struct type.
-     *
-     * @param resource   to get required details
-     * @param bJson      which needs to be converted
-     * @param structType of required struct
-     * @return converted struct
-     */
-    public static BStruct convertJSONToStruct(Resource resource, BJSON bJson, BType structType) {
-        PackageInfo packageInfo = getPackageInfo(resource);
```

Comment:
I think now "getPackageInfo()" method has only one usage. It is used in "createAndGetStruct()" in StructHelper class. Shall we move the content of "getPackageInfo()" and remove it?

Reply:
+1

Reply:
+1. 
There's another struct utils class `BLangVMStructs` which has helper methods for structs. I think the content of `StructHelper` class should also be moved to that. 
So shall we do those cleanups separately? We can do that as a part of the ongoing refactoring..

Reply:
+1

Reply:
StructHelper class is from connector directory and deals with connector related stuff. So moving it inside the org.ballerinalang.bre.bvm with all the connector-related imports might not be good. 
WDYT?

Reply:
AFAIU, `ConnectorUtils` is the connector helper class which contains connector specific stuff. And `StructHelper` is a helper to that helper, which contains only struct related operations. :)

Reply:
You are correct. :) That's because resource info is not visible to the ConnectorUtils. It gets the help from StructHelper class Which is in the same package with BResource to access it.